packages/pwa-kit-extension-sdk/src/configs/webpack/overrides-resolver-loader.ts

```diff
@@ -89,6 +90,23 @@ const OverrideResolverLoader = function (this: LoaderContext<any>) {
         ...options?.resolveOptions
     })
 
+    // Record override stats if RECORD_OVERRIDES is enabled
+    if (
+        process.env.RECORD_OVERRIDES === 'true' &&
```

Comment:
Do we really need to have this condition 'process.env.RECORD_OVERRIDES === 'true'' .. the overrideStats would only exist if this it was set by the plugin which uses that en variable. 

I wonder if it's best that we simply test for the compilation objects having the stats store indicate if we should or shouldn't record.